common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -256,7 +260,9 @@ public void run() {
                         // commandResult = sCommandResultCache.get(command);
                         //If nothing in cache, execute the command and cache the result
 //                        if (commandResult == null) {
-                            commandResult = executeCommand(command);
+                        CodeMarkerManager.getInstance().markCode(PerfConstants.CodeMarkerConstants.ACQUIRE_TOKEN_SILENT_COMMAND_EXECUTION_START);
```

Comment:
I generally think that `import static` should be used judiciously... this might be a decent place for it though...

```java
// Declare once...
final CodeMarkerManager codeMarker = CodeMarkerManager.getInstance();

codeMarker.markCode(ACQUIRE_TOKEN_SILENT_COMMAND_EXECUTION_START);
```